tests/test_directory_sync.py

```diff
@@ -8,6 +8,20 @@
 from tests.utils.fixtures.mock_directory_group import MockDirectoryGroup
 
 
+def api_directory_to_sdk(directory):
+    # The API returns an active directory as 'linked'
+    # We normalize this to 'active' in the SDK. This helper function
+    # does this conversion to make make assertions easier.
+    if directory["state"] == "linked":
+        return {**directory, "state": "active"}
```

Comment:
Yeah, I bit the bullet and just did this conversion instead of having two types.